pinot-common/src/main/java/org/apache/pinot/common/config/provider/TableCache.java

```diff
@@ -66,6 +71,8 @@ public class TableCache implements PinotConfigProvider {
   private static final String TABLE_CONFIG_PATH_PREFIX = "/CONFIGS/TABLE/";
   private static final String SCHEMA_PARENT_PATH = "/SCHEMAS";
   private static final String SCHEMA_PATH_PREFIX = "/SCHEMAS/";
+  private static final String LOGICAL_TABLE_PARENT_PATH = "/LOGICAL";
```

Comment:
Consider adding this under `/LOGICAL/TABLE` in case we want to add other logical abstractions (e.g. logical segments)